backend/amd64/proc.ml

```diff
@@ -575,6 +627,15 @@ let init () =
   end else
     num_available_registers.(0) <- 13
 
+(* CR mslater for xclerc: is this ok? *)
+(* This is not always the same as [all_phys_regs], as some physical registers
+   may not be allocatable (e.g. rbp when frame pointers are enabled). *)
+let precolored_regs () =
```

Comment:
I think it may break [that invariant](https://github.com/ocaml-flambda/flambda-backend/blob/main/backend/regalloc/regalloc_irc_state.ml#L536), but
it may be the invariant that needs updating.
Essentially, this is the same remark as for
`all_phys_regs`.